### ISSUE TITLE
error during self-uninstall

### ISSUE THREAD

**Issue by sanyer** [fisherman/fisherman#398]
Running `$ fisher self-uninstall` shows this error after warning block about files removal:

```
fish: Unknown command '2'
~/.config/fish/functions/fisher.fish (line 17):
            " &2
               ^
in function '__fisher_self_uninstall'
	called on line 201 of file ~/.config/fish/functions/fisher.fish

in function 'fisher'
	called on standard input
	with parameter list 'self-uninstall'
```

**Comment by jbucaran** [comment]
@sanyer What version of fish are you running?

**Comment by sanyer** [comment]
2.6.0

**Comment by jbucaran** [comment]
Sounds good, what version of fisherman was that?

**Comment by sanyer** [comment]
2.13.1

**Comment by jbucaran** [comment]
@sanyer Thanks for reporting. Fixed in 2.13.2. 👋

**Comment by sanyer** [comment]
@jbucaran that was fast :-)
thanks
